crates/sla/src/lib.rs

```diff
@@ -53,6 +53,8 @@ decl_storage! {
         /// Mapping from accounts of vaults/relayers to their sla score
         VaultSla get(fn vault_sla): map hasher(blake2_128_concat) T::AccountId => T::SignedFixedPoint;
         RelayerSla get(fn relayer_sla): map hasher(blake2_128_concat) T::AccountId => T::SignedFixedPoint;
+        RelayerStake get(fn relayer_stake): map hasher(blake2_128_concat) T::AccountId => T::SignedFixedPoint;
```

Comment:
Add a TODO here that this is duplicated and we need to refactor.

Reply:
Also perhaps this should be an unsigned fixed point?

Reply:
I'll add the todo, but it's definitely easier to keep this signed, since all math in this module uses signed math. Storing as unsigned would mean we have to convert it to signed every time we want to use it